test/controllers/homepages_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+describe HomepagesController do
+  # it "must be a real test" do
+  #   flunk "Need real tests"
+  # end
```

Comment:
Looks like this was missed. In addition to the nominal case, the media spotlight presents and interesting edge case where there are no works in the database.